src/loaders/common/loader_helper.py

```diff
@@ -289,6 +292,41 @@ def start_podman_service(uid: int):
     return proc
 
 
+def get_containers_config():
+    """Get containers.conf file at home directory."""
+    home = os.path.expanduser("~")
+    conf_path = os.path.join(home, CONTAINERS_CONF_PATH)
+    config = configparser.ConfigParser()
+    config.read(conf_path)
+    return config, conf_path
+
+
+def is_config_modification_required():
+    """check if the config requires modification."""
+    config, _ = get_containers_config()
+    if not config.has_section("containers"):
+        return True
+    for key, val in CONTAINERS_CONF_PARAMS.items():
+        if config.get("containers", key, fallback=None) != val:
+            return True
+    return False
+
+
+def setup_callback_server_logs():
+    """Set up containers.conf file for the callback server logs."""
+    config, conf_path = get_containers_config()
+    if not config.has_section("containers"):
+        config.add_section("containers")
+
+    for key, val in CONTAINERS_CONF_PARAMS.items():
+        if config.get("containers", key, fallback=None) != val:
+            config.set("containers", key, val)
```

Comment:
```suggestion
        config.set("containers", key, val)
```
Another case where it's no slower to just set it and the outcome is the same

Reply:
Updated!

Reply:
:+1: